src/libs/actions/Task.js

```diff
@@ -200,6 +207,7 @@ function completeTask(taskReportID, taskTitle) {
             value: {
                 stateNum: CONST.REPORT.STATE_NUM.OPEN,
                 statusNum: CONST.REPORT.STATUS.OPEN,
+                lastMessageText: taskLastMessageText,
```

Comment:
Don't we need the same logic for CreateTask, ReopenTask, EditTask and CancelTask. All seems to set the `lastMessageText` but never reset it in case of failure. Or would you rather handle this separately?

Reply:
> Or would you rather handle this separately?

That was the case! But I didn't stop to think about this again after I came up with this solution, since it's pretty simple I'll see if we can do the same for the other calls : )

Reply:
Added that to ReopenTask, EditTask and CancelTask (but not to CreateTask)

Reply:
Oh, I think I added a bug in this last commit!

Reply:
All right, I fixed it! Re-tested and things seem ok